src/views/v2/month/mobile-events/mobile-day/mobile-event/title.php

```diff
@@ -22,10 +22,10 @@
 <h3 <?php tribe_classes( $classes ); ?>>
 	<a
 		href="<?php echo esc_url( $event->permalink ) ?>"
-		title="<?php echo esc_attr( get_the_title( $event->ID ) ) ?>"
+		title="<?php echo esc_attr( $event->title ) ?>"
 		rel="bookmark"
 		class="tribe-events-calendar-month-mobile-events__mobile-event-title-link tribe-common-anchor"
 	>
-		<?php echo wp_kses_post( get_the_title( $event->ID ) ); ?>
+		<?php echo $event->title; ?>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$event'.

<sub>hash: f73838050c8bcbc27d7731c63c18238a</sub>